### DISCUSSION TITLE
What is the major difference between the following repositories CoreWcf and WCF Core?

### DISCUSSION THREAD

**Discussion by amarendra-k** [CoreWCF/CoreWCF#461] [Q&A] [answered]
How different is the following repository with CoreWCF and both are from .Net foundation?
https://github.com/dotnet/wcf

**Comment by Kralizek** [comment, marked as answer]
`dotnet/wcf` contains the code and produces assemblies needed to access WCF services (ie clients).

`corewcf/corewcf` contains the code and produces assemblies to host WCF services in the .NET Core (now simply called .NET) runtime.

Also:
WCF on .NET Framework is produced and supported by Microsoft.
CoreWCF is a port of WCF on .NET Core and is produced by the open source community
